algorithms/spot_prediction/reeke.py

```diff
@@ -303,15 +297,31 @@ def _p_limits(self):
         assert abs(sin_theta) <= 1.0  # sanity check
         sin_2theta = math.sin(2.0 * math.asin(sin_theta))
 
+        # FIXME
```

Comment:
I believe this is correct, though messy. The signs of `dp_end` and `dp_beg` will always be the same

Reply:
so it was unnecessary in the original code to redefine the `sign`